rules/falco-deprecated_rules.yaml

```diff
@@ -15,23 +15,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
-# Information about rules tags and fields can be found here: https://falco.org/docs/rules/#tags-for-current-falco-ruleset
-# The initial item in the `tags` fields reflects the maturity level of the rules introduced upon the proposal https://github.com/falcosecurity/rules/blob/main/proposals/20230605-rules-adoption-management-maturity-framework.md
-# `tags` fields also include information about the type of workload inspection (host and/or container), and Mitre Attack killchain phases and Mitre TTP code(s)
+# Information about rules tags and fields can be found here:
+# https://falco.org/docs/rules/#tags-for-current-falco-ruleset
+# The initial item in the `tags` fields reflects the maturity level of
+# the rules introduced upon the proposal
+# Proposal: Rules Adoption Management Maturity Framework
+# URL: https://github.com/falcosecurity/rules/blob/main/
+# proposals/20230605-rules-adoption-management-maturity-framework.md
+# `tags` fields also include information about the type of
+# workload inspection (host and/or container), and Mitre Attack killchain
+# phases and Mitre TTP code(s)
 # Mitre Attack References:
 # [1] https://attack.mitre.org/tactics/enterprise/
-# [2] https://raw.githubusercontent.com/mitre/cti/master/enterprise-attack/enterprise-attack.json
+# [2] https://raw.githubusercontent.com/mitre/cti/master/
```

Comment:
Is it necessary to split up links to different lines?

Reply:
@incertum we have constraint for line-length <= 80 chars. so i have to split them